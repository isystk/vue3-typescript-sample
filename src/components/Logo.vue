<template>
  <v-toolbar-title>
    sample
  </v-toolbar-title>
</template>

<script lang="ts"></script>