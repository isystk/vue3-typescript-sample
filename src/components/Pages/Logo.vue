<template>
  <router-link :to="Url.TOP">
    <v-toolbar-title class="w-64"> vue3-typescript-sample </v-toolbar-title>
  </router-link>
</template>

<script setup lang="ts">
// import { useI18n } from 'vue-i18n'
// const { t } = useI18n()
import { Url } from '@/constants/url'
</script>
