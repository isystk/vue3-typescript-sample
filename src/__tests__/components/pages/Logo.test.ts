import { mount } from '@vue/test-utils'
// import { router } from '@/router'
import Logo from '@/components/pages/Logo.vue'

test('Logo', () => {
  const wrapper = mount(Logo, {
    global: {
      // plugins: [router],
    },
  })
  expect(wrapper.element).toMatchSnapshot()
})
