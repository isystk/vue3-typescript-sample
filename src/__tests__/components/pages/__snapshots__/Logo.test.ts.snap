// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo 1`] = `
<router-link
  to="/"
>
  <v-toolbar-title
    class="w-64"
  >
     vue3-typescript-sample 
  </v-toolbar-title>
</router-link>
`;
